File: Two/medium.php
<?php

require_once 'Plan.php';

class Medium extends Plan {
    public function __construct(float $price) {
        parent::__construct("Medium", $price);
    }

    public function getServices(): array {
        return ["Access to all exercise machines + " . "Group classes"];
    }

    public function calculateCost(): float {
        return $this->getPrice();
    }
}

?>

File: java/Two/Two/Medium.java
public class Medium extends Plan {
    public Medium(float price) {
        super("Medium", price);
    }

    @Override
    public String[] getServices() {
        return new String[]{"Access to all exercise machines + Group classes"};
    }

    @Override
    public float calculateCost() {
        return getPrice();
    }
}
